README.md

```diff
@@ -136,12 +90,73 @@ spec:
 EOF
 ```
 
-### Configure Issuer
+By default, the Google CAS Issuer controller will be deployed into the `cert-manager` namespace.
 
-Create a root CA from the Google dashboard, or other API - refer to the
-[official documentation](https://cloud.google.com/certificate-authority-service/docs/creating-certificate-authorities)
+```shell
+NAME                                      READY   STATUS    RESTARTS   AGE
+cert-manager-6cd8cb4b7c-m8q4k             1/1     Running   0          34h
+cert-manager-cainjector-685b87b86-4jvtb   1/1     Running   1          34h
+cert-manager-webhook-76978fbd4c-rrx85     1/1     Running   0          34h
+google-cas-issuer-687685dc46-lrjkc        1/1     Running   0          28h
+```
+
+### Setting up Google Cloud IAM
 
-Create an Issuer or ClusterIssuer:
+Firstly, create a Google Cloud IAM service account. This service account will be used by the CAS Issuer to access the Google Private CA APIs.
+
+```shell
+gcloud iam service-accounts create my-sa
+```
+
+Apply the appropriate IAM bindings to this account. This example permits the least privilege, to create  certificates (ie `privateca.certificates.create`) from a specified suboordinate CA (`my-sub-ca`), but you can use other roles as necessary (see [Predefined Roles](https://cloud.google.com/certificate-authority-service/docs/reference/permissions-and-roles#predefined_roles) for more details).
+
+```shell
+gcloud beta privateca subordinates add-iam-policy-binding my-sub-ca --role=privateca.certificateRequester --member='serviceAccount:my-sa@project-id.iam.gserviceaccount.com'
+```
+
+You can now create a service account key and download it to a local JSON file.
+
+```shell
+gcloud iam service-accounts keys create project-name-keyid.json --iam-account my-sa@project-id.iam.gserviceaccount.com
+```
```

Comment:
Should this be moved to the Outside GKE or unrelated GCP project section?

Reply:
Good spot - fixed in https://github.com/jetstack/google-cas-issuer/pull/7/commits/9d515429cd8181a94ab038ee840009a531287331